great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -280,8 +280,10 @@ def _build_numeric_columns_rule() -> Rule:
 
         # Step-2: Declare "ParameterBuilder" for every metric of interest.
 
-        column_histogram_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.get_column_histogram_metric_multi_batch_parameter_builder(
-            json_serialize=True
+        column_partition_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.build_partition_parameter_builder(
```

Comment:
What exactly makes `column.partition` an improvement over using `column.histogram` for continuous data? On the face of it, it would seem like they do the same thing.